src/Core/src/Platform/Android/MauiShapeableImageView.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Android.Content;
+using Android.Runtime;
+using Android.Util;
+using Google.Android.Material.ImageView;
+
+namespace Microsoft.Maui.Platform
+{
+	internal class MauiShapeableImageView : ShapeableImageView
```

Comment:
Just looking at this and I think that this is all Java code so we probably should just do it in Java and let the bonding do the magic.

Doing it in C# is probably going have a big perf impact.

Reply:
New issue for this Java work: https://github.com/dotnet/maui/issues/28532